### PR TITLE
[Ready For Review] - Add emptyMessage functionality in Autocomplete

### PR DESCRIPTION
**Pull request by ikashifullah**
- Add emptyMessage functionality in Autocomplete
- Fix suggested list positioning
- Update package version for npm releases
- Fix disabled state classes

### CODE REVIEW

File: libs/static/src/scss/components/_forms.scss [hunk 1]
```diff
@@ -1338,11 +1338,27 @@ label {
...
+}
+.fds-multiple-tags-below {
+  .fds-autocomplete-panel {
+    top: 56px !important;
+  }
 }
+
+.fds-multiple-tags-besides {
+  .fds-autocomplete-panel {
+    top: calc(100% - 16px) !important;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_forms.scss:1354]: @FarrukhSubhaniGH  Why we are using !important

File: libs/static/src/scss/components/_forms.scss [hunk 2]
```diff
@@ -1338,11 +1338,27 @@ label {
...
+  }
+}
+
 .fds-autocomplete-panel {
   overflow: auto;
   position: absolute;
-  left: -1px !important;
+  left: 0 !important;
```
Comment (zeenainali) [inline libs/static/src/scss/components/_forms.scss:1361]: @FarrukhSubhaniGH  Why we are using !important

**Review by zeenainali — APPROVED** [review]

File: libs/static/src/scss/components/_forms.scss (hunk 1 above)
Comment (FarrukhSubhaniGH) [reply libs/static/src/scss/components/_forms.scss:1354]: @zeenainali here we are overwriting inline-style

File: libs/static/src/scss/components/_forms.scss (hunk 2 above)
Comment (FarrukhSubhaniGH) [reply libs/static/src/scss/components/_forms.scss:1361]: @zeenainali here we are overwriting inline-style

File: libs/static/src/scss/components/_forms.scss (hunk 1 above)
Comment (zeenainali) [reply libs/static/src/scss/components/_forms.scss:1354]: @FarrukhSubhaniGH but why we are using inline styles if we have to overwrite.

**Pull request merged**